packages/documentation/src/pages/getting-started/common-tasks/new-end-to-end-test.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Write an end to end test
+tags: Nightwatch, mockapi, e2e
+---
+
+# Write an end to end test
+
+Front end engineers use end-to-end tests in `vets-website` to validate multipage applications with client-side routing. They are primarily used to assert that:
```

Comment:
Since we also have some e2e test coverage for non-form apps perhaps the first sentence should be a little more flexible here? Also, this is minor, but maybe add `(e2e)` after the first mention of `end-to-end` here for clarity in case someone have never seen "e2e" before :)

Reply:
end-to-end is good- I prefer not to use abbreviations in documentation but I'll add the the abbreviation in parenthesis.

Can you give me an example of an e2e test that doesn't focus on validating navigation? I didn't look very hard but most of the assertions are page navs. I think we usually rely on unit tests to validate the field level behavior.

Reply:
emptyPreferences.e2e.spec.js and selectedPreferences.e2e.spec.js come to mind